clmm/dataops/__init__.py

```diff
@@ -152,10 +125,7 @@ def compute_tangential_and_cross_components(
         validate_argument(locals(), "shear1", "float_array")
         validate_argument(locals(), "shear2", "float_array")
         validate_argument(locals(), "geometry", str)
-        validate_argument(locals(), "is_deltasigma", bool)
-        validate_argument(locals(), "z_lens", float, argmin=0, eqmin=True, none_ok=True)
-        validate_argument(locals(), "z_source", "float_array", argmin=0, eqmin=True, none_ok=True)
-        validate_argument(locals(), "sigma_c", "float_array", none_ok=True)
+        validate_argument(locals(), "sigma_c", "float_array")
```

Comment:
Related to the previous comment: `"float_array"` will let a list pass the validation, but it will lead to TypeError at a later stage.